modules/final_radiological_review/php/NDB_Form_final_radiological_review.class.inc

```diff
@@ -171,6 +172,13 @@ class NDB_Form_final_radiological_review extends NDB_Form
                 );
             }
 
+            if ($DB->tableExists('radiology_review') === false) {
```

Comment:
@driusan @Jkat 

I agree with you @driusan , but the reason why it worked when I tested it is because the error the user will get comes first from the `NDB_Menu_Filter_final_radiological_review.class.inc` file